Exercise_D and E/index.html

```diff
@@ -9,14 +9,41 @@
     <link rel="stylesheet" href="./assets/style/index.css" />
     <link
       rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      href=""
       integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
       crossorigin="anonymous"
     />
   </head>
   <body>
-    <!-- CODE HERE -->
+    <nav>
+      <img src="assets/images/logo.png">
+    </nav>
 
     <script src="" async defer></script>
-  </body>
+  </nav>
+
+  <div class="container">
+    <img src="assets/images/flowershop.jpg">
+    <div class="titleflowers"><h1>Beautiful Flowers</h1>
+      <p><br><button class="button">Learn more</button></p>
+    </div>
+  </div>
+
+  <div class="grid-container">
+    <div class="item1"><img class="item" src="assets/images/flower-icon.png" height="100px"></div>
```

Comment:
You can apply the style through the stylesheet this way you can keep you html code and css separate